[Common] SectionTitle 추가 및 min-height 설정

ingbinsee opened the pull request:
## 🔍️ 이 PR을 통해 해결하려는 문제가 무엇인가요?

> 어떤 기능을 구현한건지, 이슈 대응이라면 어떤 이슈인지 PR이 열리게 된 계기와 목적을 Reviewer 들이 쉽게 이해할 수 있도록 적어 주세요
> 일감 백로그 링크나 다이어그램, 피그마를 첨부해도 좋아요

- 전체페이지 SectionTitle 적용
- 데이터 양이 적어 디바이스 높이보다 콘텐츠의 높이가 낮은 문제 해결

## ✨ 이 PR에서 핵심적으로 변경된 사항은 무엇일까요?

> 문제를 해결하면서 주요하게 변경된 사항들을 적어 주세요

- 공통으로 사용하기 위해 작업했던 SectionTitle 적용
- height: 100vh를 적용하여 페이지 높이 조정

## 🔖 핵심 변경 사항 외에 추가적으로 변경된 부분이 있나요?

> 없으면 "없음" 이라고 기재해 주세요

- 없음

## 🙏 Reviewer 분들이 이런 부분을 신경써서 봐 주시면 좋겠어요

> 개발 과정에서 다른 분들의 의견은 어떠한지 궁금했거나 크로스 체크가 필요하다고 느껴진 코드가 있다면 남겨주세요

- 없음

## 🩺 이 PR에서 테스트 혹은 검증이 필요한 부분이 있을까요?

> 테스트가 필요한 항목이나 테스트 코드가 추가되었다면 함께 적어주세요

- 없음

### 📌 PR 진행 시 이러한 점들을 참고해 주세요

- Reviewer 분들은 코드 리뷰 시 좋은 코드의 방향을 제시하되, 코드 수정을 강제하지 말아 주세요.
- Reviewer 분들은 좋은 코드를 발견한 경우, 칭찬과 격려를 아끼지 말아 주세요.
- Review는 특수한 케이스가 아니면 Reviewer로 지정된 시점 기준으로 3일 이내에 진행해 주세요.
- Comment 작성 시 Prefix로 P1, P2, P3 를 적어 주시면 Assignee가 보다 명확하게 Comment에 대해 대응할 수 있어요
  - P1 : 꼭 반영해 주세요 (Request Changes) - 이슈가 발생하거나 취약점이 발견되는 케이스 등
  - P2 : 반영을 적극적으로 고려해 주시면 좋을 것 같아요 (Comment)
  - P3 : 이런 방법도 있을 것 같아요~ 등의 사소한 의견입니다 (Chore)

#

---

## 📝 Assignee를 위한 CheckList

- [ ] To-Do Item

ingbinsee committed:
✨ [Feat] : SectionTitle 추가

전체 페이지 SectionTitle을 추가하였습니다.

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

ingbinsee committed:
🌈 [Design] : SectionTitle x축 padding 설정

SectionTitle x축에 padding 16px(모바일), 64px(태블릿, 데스크탑)을 설정하였습니다.

Related to: #108

ryudg commented on src/components/Team/RoleSection.jsx:
```diff
@@ -12,8 +12,8 @@ function RoleSection({ data, state }) {
-    <ul>
-      <li className={styles.roleSectionWrapper}>
+    <ul className={styles.roleSectionWrapper}>
+      <li className={state !== '전체' && 'h-[calc(100vh-284.37px)]'}>
```
**P2**: `layout/RootLayout.jsx` 컴포넌트에 `min-h-screen` className을 이용해서 높이값을 조정해 보세요

ingbinsee replied:
페이지 내 코드에서 수정해야 한다는 편협한 생각을 했었네요. 감사합니다.
리뷰 참고하여 코드 수정하였습니다.

ryudg reviewed (approved):
`src/components/Team/RoleSection.jsx` 이외에 특이사항 없습니다!

ingbinsee committed:
🐛 [Fix] : RootLayout height 설정

데이터의 양이 적어 디바이스 높이보다 콘텐츠의 높이가 낮은 문제가 발생함에 따라 RootLayout에 min-height: 100vh를 설정하였습니다.

Related to: #118, #120

dani-day reviewed (approved):
이견 없습니다 :)
